lib/iris/common/resolve.py

```diff
@@ -76,6 +331,8 @@ def __call__(self, lhs, rhs):
         self._metadata_mapping()
         self._metadata_prepare()
 
+        return self
+
```

Comment:
Returning `self` supports method chaining after making a call to a `Resolve` instance, which I wanted to do in the doc-tests e.g., now supports such patterns like
```python
resolve = Resolve()
resolve(c1, c2).map_rhs_to_lhs
```
which is pretty handy for certain workflows